apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/oversendelsebrev/OversendelseBrevService.kt

```diff
@@ -181,19 +181,16 @@ class OversendelseBrevServiceImpl(
         personerISak: PersonerISak,
     ): Mottaker =
         with(personerISak) {
-            val mottakerFnr: String? =
-                when (verge) {
-                    is Vergemaal -> verge.foedselsnummer.value
-                    is UkjentVergemaal -> null
+            when (verge) {
+                is Vergemaal -> tomMottaker()
```

Comment:
Sikker en god grunn til det, men er det en grunn til at denne ikke her tilsvarende den over:
`is Vergemaal -> tomMottaker().copy(foedselsnummer = MottakerFoedselsnummer(verge.foedselsnummer.value))`